articles/specs/launch.0.1.1.xsd

```diff
@@ -475,7 +477,7 @@
           </xs:documentation>
         </xs:annotation>
       </xs:attribute>
-      <xs:assert test="(@name and not(@from)) or (not(@name) and @from)"/>
+      <!-- <xs:assert test="(@name and not(@from)) or (not(@name) and @from)"/> -->
```

Comment:
Why are we commenting this out?

Reply:
Shouldn't be. See https://github.com/ros2/design/pull/278#discussion_r409649671.